### PR TITLE
Steph

### PR DESCRIPTION
**Pull request by jordan-cutler**
Many UI changes. still some fixing needs to be done but its good enough to deploy.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <!--<link href="assets/stylesheets/bootstrap.min.css" rel="stylesheet">-->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
```
Comment (lehighjcut) [inline index.html:33]: I noticed we use screaming snake case in other parts of the code. Should we match that here?

Comment (lehighjcut) [inline index.html:33]: Can we move these to the bottom of the file?

Comment (lehighjcut) [inline index.html:33]: Let's move the logic for this to a separate component so each just has a single responsibility

Comment (lehighjcut) [inline index.html:33]: Since this component is getting large, what do you think about splitting this part into a separate component?

Comment (lehighjcut) [inline index.html:33]: nit: should we move this into a constant?

Comment (jordan-cutler) [inline index.html:33]: can we move this into a constant?

**Pull request merged**